docs/development.md

```diff
@@ -112,7 +112,7 @@ INFO     (server): Listening on http://localhost:3000
 To automatically configure your GitHub App, follow these steps:
 
 1. Run the app locally by running `npm start` in your terminal.
-1. Next follow instructions to visit [http://localhost:3000](http://localhost:3000) (or your custom Glitch URL).
+1. Next follow instructions to visit [http://localhost:3000](http://localhost:3000) (or your app's Webhook URL).
```

Comment:
@wolfy1339 
I've made some slight changes to the wording after getting approval.

Reply:
:+1:  LGTM